backends/qualcomm/runtime/QnnManager.cpp

```diff
@@ -210,7 +210,7 @@ Error QnnManager::Compile(
     }
 
     for (const auto& param : op_wrapper->GetParams()) {
-      auto* p_tensor_param = dynamic_cast<TensorParamWrapper*>(param.get());
+      auto* p_tensor_param = static_cast<TensorParamWrapper*>(param.get());
```

Comment:
We cannot abandon `dynamic_cast` here, since the `param.get()` might be 'ScalarParamWrapper'. Looks like you need to add `-frtti` manually to the compile option in release build (I think executorch is not really antagonistic to it from document description).

Reply:
Okay~